openapi/listings/listings.yaml

```diff
@@ -149,6 +150,158 @@ paths:
+                        type: string
+                        description: The URI at which the next batch of listing profiles can be gotten from
+                        format: uri
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Produces a list of listing profiles
+      parameters:
+        - schema:
+            type: string
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: 'filter[businessPartner.id]'
+          description: Return listing profiles for the specified partner
+          required: true
+        - schema:
+            type: string
+          in: query
+          description: 'The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links. [Pagination Docs](https://developers.vendasta.com/platform/ZG9jOjEwMTkzMDg0-overview#paging).'
+          name: 'page[cursor]'
+        - schema:
+            type: string
+            maxLength: 25
+            default: '25'
+            minLength: 1
+          in: query
+          description: 'The maximum number of tasks you would like returned in a single batch. Use the links.next member in the response to get the remainder. [Pagination Docs](https://developers.vendasta.com/platform/ZG9jOjEwMTkzMDg0-overview#paging).'
+          name: 'page[limit]'
+        - schema:
+            type: string
+            enum:
+              - createdAt
+              - '-createdAt'
+              - name
+              - '-name'
+              - updatedAt
+              - '-updatedAt'
+            default: '-updatedAt'
+          in: query
+          name: sort
+          description: Sort listing profile based on selected option. Defaulted to -updatedAt (Desc updatedAt)
+      security:
+        - OAuth2Demo:
+            - business
+            - 'listings:read'
+        - OAuth2Prod:
+            - business
+            - 'listings:read'
+      tags:
+        - Listing Profiles
+      x-lifecycle:
+        status: trustedTester
+  '/listingProfiles/{id}':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+    patch:
+      summary: Update Listing Profile
+      operationId: patch-listingProfiles-id
+      tags:
+        - Listing Profiles
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Update the existing listing profile.
+        Only the root ID and type fields are required. All others are optional and will keep their original value if omitted.
+      x-lifecycle:
+        status: trustedTester
```

Comment:
We should record the status as "proposed" until we are ready for a partner to test the new operations.